packages/neo-one-website/docs/0-installation/0-quick-start.md

```diff
@@ -0,0 +1,285 @@
+// for example:
+// if you contract name is Test
+// then the function will become createTestSmartContract()
+```
+
+Check out our [Client APIs](/docs/client-apis) page for more details.
+
+---
+
+## Deploy
+
+::: warning
+
+Important
+
+If you do not know what contracts need to be deployed, check the following
+[What Contracts Need To Be Deployed?](https://docs.neo.org/docs/en-us/sc/deploy/deploy.html#what-contracts-need-to-be-deployed)
+
+:::
+
+### Checklist:
+
+- Wallet's private key
+- Test coins
+- Migration file
+
+You first need a wallet and the wallet's private key and sufficient amount of NEO and GAS to deploy.
+
+### Wallet set-up:
+
+1. Go to `https://neotracker.io/`
+2. Go to `“Wallet”` tab.
+3. Click on `“New Wallet”`.
+4. Enter a password (make sure you save this password), which is used to unlock the wallet later.
+5. Click on `"Download Encrypted Key"` (this will generate a `.txt` file that can be used to unlock your wallet via `“Keystore File”` option).
+6. Save the private key shown on screen locally and secure it.
+7. Click continue to view your wallet.
+8. (Optional) Generate a PDF by clicking on “Print paper wallet”
+9. (Optional) Scroll down to `“Details”` when viewing your wallet to access the options of the previous steps (such as to view your private key or download keystore file).
+
+### Migration File:
+
+Create a `migration.ts` (`migration.js` if using JS) under `neo-one` folder.
+
+::: warning
+
+Tip
+
+VSCode might display single child folders in "compact form" and prevent you from creating a new file under neo-one. Disable "compact folder" in "settings" if needed.
+
+:::
+
+Example:
+
+```typescript
+import { MigrationContracts } from '../src/neo-one';
+
+export default ({ token, ico, escrow }: MigrationContracts, _network: string) => {
+  token.deploy();
+};
+```
+
+::: warning
+
+Note
+
+For more details on deployment specifics and migration files, check out the [Deployment](/docs/deployment) page.
+
+:::
+
+### Get Test coins
+
+You can get test coins automatically from https://neowish.ngd.network/
+
+Limited to 1000 NEO and 1000 GAS per day.
+
+If you need more than that. You must apply through Neo website. Please follow the instructions here: https://docs.neo.org/docs/en-us/network/testnet.html#applying-for-test-coin-from-neo-website
+
+::: warning
+
+Note
+
+- It costs about 500 GAS to deploy a simple contract. Checkout [System Fees](https://docs.neo.org/docs/en-us/sc/fees.html) for more details.
+- Test coins only show on TestNet. Use https://testnet.neotracker.io to explore transactions and use the wallet on TestNet.
+
+:::
+
+### Deploy to TestNet:
+
+```bash
+yarn neo-one deploy
+```
+
+### Deploy to MainNet (or others):
+
+```bash
+yarn neo-one deploy --network [network-name]
+```
+
+Where `network` is one of the keys provided by your configuration (`.neo-one.config.ts`) under `networks` property. By default yarn neo-one deploy will use the test key.
```

Comment:
"... under the `networks` property."

"By default `neo-one deploy` will use the `test` key."